Taskfile.yml

```diff
@@ -0,0 +1,166 @@
+version: '3'
+
+tasks:
+
+  create-network:
+    desc: Create the external network
+    cmds:
+      - docker network create external-net
+
+  init-backend:
```

Comment:
After running `task clean`, would I always need to run `init-backend` before `backend-up`?

Reply:
If you need to do migrations, then yes